metrics/falcon.go

```diff
@@ -0,0 +1,48 @@
+package metrics
+
+import falcon "github.com/niean/goperfcounter"
+
+type falconGauge struct {
+	name string
+}
+
+// Add add value of counter
+func (f *falconGauge) Add(value int64) {
+	falcon.SetCounterCount(f.name, value)
+}
+
+// Inc add value of counter, value = 1
+func (f *falconGauge) Inc() {
+	f.Add(1)
+}
+
+// Decrease decrease value of counter
+func (f *falconGauge) Decrease(value int64) {
+	falcon.SetCounterCount(f.name, -value)
+}
+
```

Comment:
We do not need this API. Use `Add(-12)` if the user needs to.

Reply:
`Add(-XXX) ` , I only think it is a strange api, add(5) and decrase(5) is more clear

Reply:
ok, `Sub` is the appropriate antonym of `Add`. Rename to this.